src/status.js

```diff
@@ -1,38 +1,116 @@
-import { createStatusManager } from '@digitalcredentials/credential-status-manager-git';
+import {
+  createStatusManager as createStatusManagerDb
+} from '@digitalcredentials/credential-status-manager-db';
+import {
+  createStatusManager as createStatusManagerGit
+} from '@digitalcredentials/credential-status-manager-git';
 import { getConfig } from './config.js';
 
 const {
-    credStatusRepoName,
-    credStatusMetaRepoName,
-    credStatusOwnerAccountName,
-    credStatusAccessToken,
-    credStatusDidSeed
+  // Database env vars
+  statusCredentialSiteOrigin,
+  credStatusDatabaseUrl,
+  credStatusDatabaseHost,
+  credStatusDatabasePort,
+  credStatusDatabaseUsername,
+  credStatusDatabasePassword,
+
+  // Git env vars
+  credStatusService,
+  credStatusRepoName,
+  credStatusRepoId,
+  credStatusMetaRepoName,
+  credStatusMetaRepoId,
+  credStatusOwnerAccountName,
+  credStatusAccessToken,
+  credStatusDidSeed
 } = getConfig();
 
 let STATUS_LIST_MANAGER;
 
+async function createDatabaseStatusManager() {
+  return createStatusManagerDb({
+    statusCredentialSiteOrigin,
+    databaseService: credStatusService,
+    databaseUrl: credStatusDatabaseUrl,
+    databaseHost: credStatusDatabaseHost,
+    databasePort: credStatusDatabasePort,
+    databaseUsername: credStatusDatabaseUsername,
+    databasePassword: credStatusDatabasePassword,
+    didMethod: 'key',
+    didSeed: credStatusDidSeed,
+    signUserCredential: false,
+    signStatusCredential: true
+  });
+}
+
+async function createGitHubStatusManager() {
+  return createStatusManagerGit({
+    service: credStatusService,
```

Comment:
As a heads up, I am considering changing this to `gitService` to be consistent with `databaseService`. The other option would be to use `service` for both.

Reply:
Good point - I like the idea of switching it to gitService since that's what it is doing: telling us which git service is being used.